holidays/calendars/hindu.py

```diff
@@ -784,6 +816,36 @@ class _HinduLunisolar:
         2035: (OCT, 10),
     }
 
+    # Dates for Maharana Pratap Jayanti (Jyeshtha Shukla 3)
+    MAHARANA_PRATAP_JAYANTI_DATES = {
+        2010: (JUN, 15),
+        2011: (JUN, 4),
+        2012: (MAY, 24),
+        2013: (JUN, 11),
+        2014: (MAY, 31),
+        2015: (MAY, 20),
+        2016: (JUN, 7),
+        2017: (MAY, 28),
+        2018: (JUN, 16),
+        2019: (JUN, 6),
+        2020: (MAY, 25),
+        2021: (JUN, 13),
+        2022: (JUN, 2),
+        2023: (MAY, 22),
+        2024: (JUN, 9),
+        2025: (MAY, 29),
+        2026: (JUN, 17),
+        2027: (JUN, 7),
+        2028: (MAY, 27),
+        2029: (MAY, 16),
```

Comment:
Jun 16, according to https://www.drikpanchang.com/calendars/indian/jayanti/maharana-pratap/maharana-pratap-jayanti.html?year=2029